lib/modules/manager/ocb/__fixtures__/valid.yaml

```diff
@@ -0,0 +1,16 @@
+dist:
+  name: otelcol-custom
+  description: Local OpenTelemetry Collector binary
+  module: github.com/open-telemetry/opentelemetry-collector
+  otelcol_version: 0.40.0
+  version: 1.0.0
+  output_path: /tmp/dist
+exporters:
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.86.0
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.86.0
+
+receivers:
+  - gomod: go.opentelemetry.io/collector/receiver/otlpreceiver v0.86.0
+
+processors:
+  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.86.0
```

Comment:
Can we inline this?

Reply:
We can, tough this fixture is quite big for my taste.